packages/swift/build.sh

```diff
@@ -126,6 +123,7 @@ termux_step_pre_configure() {
 }
 
 termux_step_make() {
+	ln -sf $TERMUX_PKG_HOSTBUILD_DIR/llvm-linux-x86_64 $TERMUX_PKG_BUILDDIR/llvm-linux-x86_64
```

Comment:
This will work but move it inside the on-device build check below, as it isn't needed on-device.

Reply:
Fixed in https://github.com/termux/termux-packages/pull/7376/commits/7001b64eff32c8933f8dd96e6d1325268acdcdcd, build finishes without error now